pyproject.toml

```diff
@@ -1,2 +1,30 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "mona-openai"
+version = "0.0.3"
```

Comment:
Not really important - but maybe there's a way to delete the previous bad versions such that our first working version will be 0.0.1?

Reply:
There isn't. Checked.